agreement_legal/__manifest__.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2018 - TODAY, Pavlov Media
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Agreements Legal",
+    "summary": "Manage Agreements, LOI and Contracts",
+    "author": "Pavlov Media, "
+              "Open Source Integrators, "
+              "Yves Goldberg (Ygol Internetwork), "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/contract",
+    "category": "Partner",
+    "license": "AGPL-3",
+    "version": "12.0.1.0.0",
+    "depends": [
+        "account",
```

Comment:
Remove a **product, mail, and account** modules from dependency.

below is the dependency of a sale_management module.
sale_management << sale and digest
sale << sales_team, payment and portal
payment << account
account << base_setup, product, analytic, portal and digest
product << base, deciamal_precision, mail and uom

Reply:
unless new consensus; we're migrating agreement (v11) as it is and we will refactor after the migration merge.

Reply:
@ygol We can change it in the migration part.

Reply:
@murtuzasaleh @max3903 

I will remove the fields:

- `sale_order_id`
- `order_lines_services_ids`
- **total_company** and **customer mrc** & **nrc**
- `analytic_id` & `analytic_line_ids`
- the file "**models/account.py**"

- what about `serviceprofile_ids` and the file "**models/serviceprofile.py**" ?

Is it ok to keep products (`line_ids`) ? I use it and it would be nice not to have to install "sales"

To what correspond the "**mail**" dependency?

TIA

Reply:
Service profile related code can go into a separate new module (agreement_serviceprofile).

It is ok to keep line_ids.

We have a dependency on mail because of the chatter on agreement.

Reply:
> Service profile related code can go into a separate new module (agreement_serviceprofile).

@max3903 So in v12 agreement_serviceprofile module will create based on agreement module (v11).
Can I work on that module? Because it is used in (agreement_maintenance, agreement_stock, agreement_rma, agreement_mrp, agreement_repair, agreement_helpdesk) modules.

@ygol Have you started `agreement_project` module migration, If not can I start working on it?

Reply:
@murtuzasaleh Sure.

Reply:
@max3903 And for `agreement_project` module migration? Can I start working on it?

Reply:
@murtuzasaleh Please refer to the plan here: https://github.com/OCA/contract/issues/189